src/app/components/elements/views/ProgressBar.tsx

```diff
@@ -10,19 +10,8 @@ interface ProgressBarProps {
     type?: string;
 }
 
-const BAR_COLOURS = new Map([
-    ["phys_book_step_up", "bar-gcse"],
-    ["phys_book_gcse", "bar-gcse"],
-    ["physics_skills_14", "bar-alevel"],
-    ["physics_skills_19", "bar-alevel"],
-    ["physics_linking_concepts", "bar-alevel"],
-    ["maths_book_gcse", "bar-maths"],
-    ["maths_book", "bar-maths"],
-    ["chemistry_16", "bar-chemistry"]
-]);
-
 export const ProgressBar = ({percentage, primaryTitle, secondaryPercentage, secondaryTitle, children, type}: ProgressBarProps) => {
```

Comment:
It looks like `type` is not being set at the moment anywhere in the whole codebase, so you could rename it to `colour` and that way you can get rid of the `colour` constant underneath without breaking any functionality.